### PR TITLE
Fix encoder bug zoom65 and zoom65_litte

### PR DESCRIPTION
**Pull request by spbgzh**
## Description
Fix encoder bug zoom65 and zoom65_litte

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR
https://github.com/qmk/qmk_firmware/pull/15218
https://github.com/qmk/qmk_firmware/pull/14926

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/meletrix/zoom65/config.h [hunk 1]
```diff
@@ -42,6 +42,12 @@
 #define ENCODERS_PAD_B { B0 }
 
 #define ENCODERS 1
+
+#ifdef ENCODER_RESOLUTION
+    #undef ENCODER_RESOLUTION
+#endif
```
Comment (zvecr) [inline keyboards/meletrix/zoom65/config.h:48]:
```suggestion
```

File: keyboards/meletrix/zoom65_lite/config.h [hunk 2]
```diff
@@ -42,6 +42,12 @@
 #define ENCODERS_PAD_B { B0 }
 
 #define ENCODERS 1
+
+#ifdef ENCODER_RESOLUTION
+    #undef ENCODER_RESOLUTION
+#endif
```
Comment (zvecr) [inline keyboards/meletrix/zoom65_lite/config.h:48]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/meletrix/zoom65/config.h (hunk 1 above)
Comment (spbgzh) [reply keyboards/meletrix/zoom65/config.h]: Deleted

**Review by zvecr — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by ibrokemypie** [comment]
Which product is the zoom65_lite? Can't find any mention of it online

Do either of these cover the zoom65 essential edition (wireless version)?

**Comment by BigBoiMan123** [comment]
Will the zoom65 v2 be added?

**Pull request merged**